### PR TITLE
add sqlparse<0.2

### PR DESCRIPTION
**Pull request by Mariatta**
added sqlparse<0.2 to requirements.txt to address https://github.com/python/pythondotorg/issues/989

### CODE REVIEW

**Comment by berkerpeksag** [comment]
Could you add a TODO comment and explain why we need to add this (something like "delete this when we switch to Django 1.8 and DDT 1.5")?

File: requirements.txt [hunk 1]
```diff
@@ -43,3 +43,4 @@ uWSGI==2.0.10
 
 raven==5.2.0
 django-waffle==0.11.1
+sqlparse <0.2
```
Comment (berkerpeksag) [inline requirements.txt]: Nitpick: sqlparse <0.2 -> sqlparse<0.2

**Comment by Mariatta** [comment]
Thanks for the feedback @berkerpeksag 
Added todo comment and removed the space before < sign

**Comment by berkerpeksag** [comment]
Thanks!

**Pull request merged**